stream_alert/classifier/payload/apps.py

```diff
@@ -0,0 +1,45 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from stream_alert.classifier.payload.payload_base import (
+    PayloadRecord,
+    RegisterInput,
+    StreamPayload
+)
+from stream_alert.shared import CLASSIFIER_FUNCTION_NAME as FUNCTION_NAME
+from stream_alert.shared.metrics import MetricLogger
+
+
+@RegisterInput
```

Comment:
no idea how this works, would love to learn

Reply:
let's sync in person about this - essentially this class decorator acts as a factory to register the payload classes for lookups